src/screens/profileView/profileViewStyles.ts

```diff
@@ -0,0 +1,132 @@
+import { Input, inputClasses } from '@mui/base/Input';
+import { styled } from '@mui/material';
+import { Box, Button, IconButton, ListItem } from '@mui/material';
+
+import { theme } from '../../assets/styles/theme';
+
+export const Wrapper = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  padding: 32px;
+  box-sizing: border-box;
+  position: relative;
+  font-family: 'Inter', sans-serif;
+  @media (max-width: 600px) {
+    padding: 16px;
+  }
+`;
+
+export const EditButton = styled(IconButton)`\
+    border-radius: 9999px;
+    border: 2px solid ${theme.colors.grey_25};
+      background-color: ${theme.colors.white_100};
+  @media {
+    color: ${theme.colors.black_100};
+  }
+`;
+
+export const EditButtonWrapper = styled(Box)`
+  display: flex;
+  position: absolute;
+  right: 2rem;
+  top: 6rem;
+  margin-right: auto;
+  justify-content: flex-start;
+`;
+
+export const AvatarWrapper = styled(Box)`
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0.5rem;
+  font-size: 24px;
+`;
+
+export const ChangePasswordButton = styled(Button)`
+  background-color: ${theme.colors.red_100};
+  color: ${theme.colors.white_100};
+  &:hover {
+    background-color: ${theme.colors.red_100};
+  }
+  margin: 10px 0 15px;
+  padding: 16px 0;
+  font-size: 18px;
+  border-radius: 8px;
+  width: 90%;
+  @media (max-width: 600px) {
+    margin: 5px 0 10px;
+    padding: 12px 0;
+    font-size: 16px;
+  }
+`;
+
+export const Item = styled(ListItem)`
+  position absolute;
+  background-color: ${theme.colors.red_25};
+  border-radius: 12px;
+  margin-bottom: 8px;
+`;
+
+export const StyledInput = styled(Input)(`
+  display: inline-block;
+
+  .${inputClasses.input} {
+    width: 18.5rem;
+    font-family: 'IBM Plex Sans', sans-serif;
```

Comment:
zmień font na `'Inter', sans-serif;`

Reply:
Odnośnie tego wcześniejszego żeby zmienić wielkości jak są na dc to jakbym miał zmienić w elementach strony na padding: 0 2rem to nie wygląda to dobrze w przypadku niektórych elementów jak sprawdzałem